crates/evm/traces/src/inspector.rs

```diff
@@ -128,32 +123,25 @@ impl Tracer {
         step.gas_cost = step.gas - interp.gas.remaining();
 
         // Error codes only
-        if status as u8 > InstructionResult::OutOfGas as u8 {
-            step.error = Some(format!("{status:?}"));
+        if interp.instruction_result as u8 > InstructionResult::OutOfGas as u8 {
+            step.error = Some(format!("{:?}", interp.instruction_result));
```

Comment:
relying on u8 cmp is pretty bad, this can be `if .is_err()`

Reply:
yeah, this predates fn is_error, updated